.github/workflows/sycl-linux-precommit.yml

```diff
@@ -96,7 +74,19 @@ jobs:
             target_devices: level_zero:gpu;opencl:gpu
             reset_intel_gpu: true
             extra_lit_opts: --param matrix-xmx8=True
-            env: '{"LIT_FILTER":${{ needs.determine_arc_tests.outputs.arc_tests }} }'
```

Comment:
Workflow invocation at line 113 didn't actually pass `env` to the reusable workflow, so we were already testing all the e2e tests in this mode.